lib/lib.go

```diff
@@ -0,0 +1,24 @@
+package lib
+
+import (
+	"encoding/gob"
+	"os"
+
+	"github.com/bits-and-blooms/bloom/v3"
+)
+
+// BloomFilterFromFile opens a .gob file and returns a bloom.BloomFilter
+func BloomFilterFromFile(filename string) (*bloom.BloomFilter, error) {
```

Comment:
why not just use the bloom.BloomFilter's ReadFrom function?

Reply:
1. I don't think `ReadFrom` works as I use gob.NewEncoder ... then 
```
encoder := gob.NewEncoder(fname); 
```
```
encoder.Encode(filter);
```

2. NewDecoder already has buff

decoder := gob.NewDecoder(file) <-- this seems wrapped a by buffer

```
NewDecoder returns a new decoder that reads from the [io.Reader](https://pkg.go.dev/io#Reader). If r does not also implement [io.ByteReader](https://pkg.go.dev/io#ByteReader), it will be wrapped in a [bufio.Reader](https://pkg.go.dev/bufio#Reader).
```

Reply:
I think the gob.NewEncoder actually use the encoder in bloom package anyway.  But there might be some slight difference in the wrapper.  This is not a significant issue.